R/parameter-range.R

```diff
@@ -11,32 +11,33 @@ ParameterRange <- R6::R6Class(
   active = list(
     #' @field min Minimum value for the parameter range
     min = function(value) {
-      private$wrapProperty("Min", value)
+      private$.wrapProperty("Min", value)
     },
     #' @field max Maximum value for the parameter range
     max = function(value) {
-      private$wrapProperty("Max", value)
+      private$.wrapProperty("Max", value)
     },
     #' @field unit Unit in which the value is defined
     unit = function(value) {
-      private$wrapProperty("Unit", value)
+      private$.wrapProperty("Unit", value)
     }
   ),
   public = list(
     #' @description
     #' Initialize a new instance of the class
-    #' @param ref Optional .NET reference object. If not defined, a new instance will be created
+    #' @param netObject Optional `NetObject` of `ParameterRange`. If not defined, a new instance will be created
     #' @param min Optional minimum value for the range
     #' @param max Optional minimum value for the range
     #' @param unit Optional unit of the specified min and max
     #' @return A new `ParameterRange` object.
-    initialize = function(ref = NULL, min = NULL, max = NULL, unit = NULL) {
+    initialize = function(netObject = NULL, min = NULL, max = NULL, unit = NULL) {
       validateIsNumeric(min, nullAllowed = TRUE)
       validateIsNumeric(max, nullAllowed = TRUE)
       validateIsString(unit, nullAllowed = TRUE)
-      ref <- ref %||% rClr::clrNew("PKSim.Core.Snapshots.ParameterRange")
-      super$initialize(ref)
+      netObject <- netObject %||% rSharp::newObjectFromName("PKSim.Core.Snapshots.ParameterRange")
+      super$initialize(netObject)
       # Because of weird issue with nullable value in rClr
+      # https://github.com/Open-Systems-Pharmacology/OSPSuite-R/issues/1369
       if (!is.null(min)) {
         self$min <- min
       }
```

Comment:
The comment refers to rClr.
Is just the comment obsolete or the code as well?

Reply:
referenced issue says "reevaluate after rDharp migration". let's put a TODO in the comment here so that we can easily find it later